### PR TITLE
Create MySQL datastore in AML

### PR DESCRIPTION
**Pull request by achalachal2003**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

**Comment by achalachal2003** [comment]
@bmoore-msft for review, thanks!

File: 101-datastore-create-mysql/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+		"port": {
+			"type": "string",
+			"defaultValue": "3306",
+			"metadata": {
+				"description": "Optional : The port number. Defaults to 3306"
+			}
+		},
+		"endpoint": {
+			"type": "string",
+			"defaultValue": "mysql.database.azure.com",
```
Comment (bmoore-msft) [inline 101-datastore-create-mysql/azuredeploy.json:50]: see comments in other PRs and make the same changes here

**Comment by bmoore-msft** [comment]
@achalachal2003 - can you check the merge conflicts

**Comment by achalachal2003** [comment]
> @achalachal2003 - can you check the merge conflicts

Fixed, this branch got messed up.

**Pull request merged**